.github/workflows/nodejs.yml

```diff
@@ -186,15 +186,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-cache-
       - run: yarn install --frozen-lockfile
-      - uses: actions/cache@v2
```

Comment:
why are we rm'ing this?

Reply:
When we run `check:builds` we should be rebuilding and not checking an older cached version

Reply:
At least for now until we make sure the builds are caching properly